misc/atrium-specs/src/commonMain/kotlin/ch/tutteli/atrium/specs/integration/AbstractIterableNotToContainEntriesExpectationsTest.kt

```diff
@@ -0,0 +1,294 @@
+                                .format(notToHaveElementsOrNoneFunNameSpec)
+                        }"
+                    )
+                }
+            }
+        }
+
+    @TestFactory
+    fun iterable__happy_case() =
+        nonNullableCases(notToContainEntriesSpec, notToContainNullableEntriesSpec) { notToContainFunArr ->
+            fun Expect<Iterable<Double>>.notToContainFun(
+                a: Expect<Double>.() -> Unit,
+                vararg aX: Expect<Double>.() -> Unit,
+            ) = notToContainFunArr(a, aX)
+            testFactory(notToContainEntriesSpec) {
+                it("$toBeGreaterThanFun(1.0) and $toBeLessThanFun(2.0) does not throw") {
+                    expect(oneToSeven()).notToContainFun({ toBeGreaterThan(1.0); toBeLessThan(2.0) })
+                }
+                it("$toEqualFun(1.1), $toEqualFun(2.2), $toEqualFun(3.3) does not throw") {
+                    expect(oneToSeven()).notToContainFun({ toEqual(1.1) }, { toEqual(2.2) }, { toEqual(3.3) })
+                }
+                it("$toEqualFun(3.3), $toEqualFun(1.1), $toEqualFun(2.2) does not throw") {
+                    ch.tutteli.atrium.api.verbs.internal.expect(oneToSeven())
+                        .notToContainFun({ toEqual(3.3) }, { toEqual(1.1) }, { toEqual(2.2) })
+                }
+            }
+            expect(oneToSeven()).notToContainFun({ toBeGreaterThan(1.0); toBeLessThan(2.0) })
+        }
+
+    @TestFactory
+    fun iterable__failing_cases_search_string_at_different_positions() {
+        nonNullableCases(notToContainEntriesSpec, notToContainNullableEntriesSpec) { notToContainFunArr ->
+            fun Expect<Iterable<Double>>.notToContainFun(
+                a: Expect<Double>.() -> Unit,
+                vararg aX: Expect<Double>.() -> Unit,
+            ) = notToContainFunArr(a, aX)
+
+            testFactory(notToContainEntriesSpec) {
+                it("$toBeLessThanFun(4.0) throws AssertionError") {
+                    expect {
+                        expect(oneToSeven()).notToContainFun({ toBeLessThan(4.0) })
+                    }.toThrow<AssertionError> {
+                        message {
+                            toContainRegex(
+                                "\\Q$rootBulletPoint\\E$notToContainDescr: $separator" +
+                                        "$indentRootBulletPoint\\Q$listBulletPoint\\E$anElementWhichNeedsDescr: $separator" +
+                                        "$afterExplanatory$toBeLessThanDescr: 4.0.*$separator" +
+                                        "$afterExplanatoryIndent\\Q$warningBulletPoint$mismatches:\\E $separator" +
+                                        "$afterMismatchedWarning${mismatchedIndex(0, "1.0")}.*$separator" +
+                                        "$afterMismatchedWarning${mismatchedIndex(1, "2.0")}.*$separator" +
+                                        "$afterMismatchedWarning${mismatchedIndex(5, "3.0")}.*"
```

Comment:
nitpick: the formatting changed here without obvious reason. Please press CTRL+L (if you use intellij) to format the file